File: DrQuine/src/Colleen.c
#include <stdio.h>
/*
    comment
*/
void nothing(void){}
int main(){char*s="#include <stdio.h>%1$c/*%1$c    comment%1$c*/%1$cvoid nothing(void){}%1$cint main(){char*s=%2$c%3$s%2$c;nothing();printf(s,10,34,s);%1$c/*%1$c    toto%1$c*/%1$c}%1$c";nothing();printf(s,10,34,s);
/*
    toto
*/
}

File: DrQuine/Bonus/Colleen.py
s = r"print('s = r\"' + s + '\"' + '\n\'\'\'com\'\'\'\ndef nothing():\n    pass\ndef main():\n    \'\'\'comment\'\'\'\n    nothing()\n    exec(s)\nmain()')"
'''com'''
def nothing():
    pass
def main():
    '''comment'''
    nothing()
    exec(s)
main()
